packages/visualizations/addon/components/navi-visualizations/table.js

```diff
@@ -201,12 +203,20 @@ export default Component.extend({
    */
   columns: computed('options.columns', 'request.sort', function() {
     let sorts = this._mapAlias(get(this, 'request')),
-        columns = $.extend(true, [], get(this, 'options.columns'));
+        columns = cloneDeep(get(this, 'options.columns') || []),
+        columnIndex = get(this, 'columnIndex');
 
-    return columns.map( column => {
+    if (!columnIndex) {
```

Comment:
Can this be it's own computed property?

Reply:
We only want to compute this value once, computed property may not be ideal